datamodels/set-up-staging.md

```diff
@@ -2,36 +2,38 @@
 
 1. Go to an emx2 server
 
-2. Make a schema called CatalogueOntologies, that will contain look-up lists.
+2. Make a schema called SharedStaging and load [shared-staging-model](https://github.com/molgenis/molgenis-py-cohorts-etl/datamodels/shared-staging-model.csv).
 
-3. Make a schema called DataCatalogue and load the [catalogue model](https://github.com/molgenis/molgenis-emx2/blob/master/data/datacatalogue/molgenis.csv)
+3. Make a schema called CatalogueOntologies. This schema will contain look-up lists. The table metadata is loaded in the next step.
+
+4. Make a schema called DataCatalogue and load the [catalogue model](https://github.com/molgenis/molgenis-emx2/blob/master/data/datacatalogue/molgenis.csv)
  that will contain all the catalogue data. The ontology_array and ontology columnTypes will instantiate tables in 
  CatalogueOntologies automatically.
 
-4. Load data in CatalogueOntologies.
+5. Load data in CatalogueOntologies.
```

Comment:
Data from: https://github.com/molgenis/molgenis-emx2/tree/master/data/datacatalogue/CatalogueOntologies or should you first load DataCatalogue (step 7) and afterwards load data in CatalogueOntologies (otherwise the lookup lists simply do not exists yet?)